src/marion/marion/certificates/manufactories/__init__.py

```diff
@@ -0,0 +1 @@
+from .dummy import DummyCertificate
```

Comment:
If I understand correctly, this module object will contain only authorized manufactories classes.
The goal is to pick them with `gettattr` in `models.Request.get_certificate_class()`.

This implementation forbid creating other attributes in this module object.
And also, it does not prevent from playing with special attributes like `__path__`, `__package__`, etc

As an alternative, I suggest to define a set of authorized manufactories class.
And to compute a dict based on this set, to be able to pick a class by name : 

```suggestion
from .dummy import DummyCertificate

manufactories = set({
  DummyCertificate,
})
manufactories_dict = dict(map(lambda m: (m.__name__, m), manufactories))
```

Reply:
I've changed this behavior to allow defining your own certificate issuers even from another Django application.